test/Main.hs

```diff
@@ -678,3 +679,9 @@ instance Arbitrary Smalldatetime where
     pure
       (Smalldatetime
          (LocalTime day (timeToTimeOfDay (secondsToDiffTime (minutes * 60)))))
+
+instance Arbitrary Datetimeoffset where
+  arbitrary = do
+    lt <- arbitrary
+    offset <- choose (-12 * 60, 14 * 60)
```

Comment:
Can you explain the logic behind these numbers in a comment?

Reply:
Certainly. Done in a new commit 😄